src/pj_internal.c

```diff
@@ -114,14 +115,15 @@ void proj_context_inherit (PJ *parent, PJ *child) {
 
 
 
-size_t pj_strlcpy(char *dst, const char *src, size_t siz) {
-/*******************************************************************
+/**************************************************************************************/
+size_t pj_strlcpy(char *dst, const char *src, size_t dsize) {
```

Comment:
Perhaps it's time to make `pj_string.c`? There's a bunch of string manipulation functions spread out across the code base. It would be nice to gather them all in the same space.

Reply:
... and while at it, move `pj_free` to `pj_malloc.c`, and perhaps renaming to `pj_memory.c`

Reply:
+1. Seperate PR though. For both I think.

Reply:
Agree